src/measure_theory/measure/measure_space.lean

```diff
@@ -837,23 +837,16 @@ have μ.to_outer_measure ≤ Inf (to_outer_measure '' m) :=
   le_Inf $ ball_image_of_ball $ λ μ hμ, to_outer_measure_le.2 $ h _ hμ,
 λ s hs, by rw [Inf_apply hs, ← to_outer_measure_apply]; exact this s
 
-instance [measurable_space α] : complete_semilattice_Inf (measure α) :=
-{ Inf_le := λ s a, measure_Inf_le,
-  le_Inf := λ s a, measure_le_Inf,
-  ..(by apply_instance : partial_order (measure α)),
-  ..(by apply_instance : has_Inf (measure α)), }
-
 instance [measurable_space α] : complete_lattice (measure α) :=
 { bot := 0,
-  bot_le := λ a s hs, by exact bot_le,
-/- Adding an explicit `top` makes `leanchecker` fail, see lean#364, disable for now
-
-  top := (⊤ : outer_measure α).to_measure (by rw [outer_measure.top_caratheodory]; exact le_top),
+  bot_le := λ a s hs, bot_le,
+  top := (⊤ : outer_measure α).to_measure $ by { rw outer_measure.top_caratheodory, exact le_top },
   le_top := λ a s hs,
-    by cases s.eq_empty_or_nonempty with h  h;
+    by cases s.eq_empty_or_nonempty with h h;
       simp [h, to_measure_apply ⊤ _ hs, outer_measure.top_apply],
--/
-  .. complete_lattice_of_complete_semilattice_Inf (measure α) }
+  ..measure.has_Inf,
+  ..measure.partial_order,
+  ..complete_lattice_of_Inf (measure α) $ λ s, ⟨λ a, measure_Inf_le, λ a, measure_le_Inf⟩ }
```

Comment:
Something I recall reading is that `complete_lattice_of_Inf` can give bad defeq properties? As such, my only question is "is this a concern here": e.g. what is the definition of Sup, inf, sup that this infers, and is it defeq to what you would expect it to be, etc.? Basically whenever using the `complete_lattice_of_*`, it's worth being careful about the data fields (the proofs I assume are fine).

Reply:
Assuming I didn't make a mistake, I didn't actual change the def-eq definitions of any of these instances. Exactly the same fields are filled out by `complete_lattice_of_Inf` as they were before by `complete_lattice_of_complete_semilattice_Inf`.

Reply:
Yeah, I just don't know if that's an issue... (it may have been a problem before).